review-responses.tex

```diff
@@ -0,0 +1,383 @@
+    should have a more dedicated repository for users to access the software and
+    start running their own simulations.
+\end{point}
+
+\reply{
+    As the reviewer suggests, our main goal here is to provide easy access
+    to the \emph{algorithm}, not to a forward simulation model --
+    we aren't describing a forward simulation model, 
+    but rather a way to speed up and get more information out of an existing one.
+    But, is the algorithm accessible?
+    An interface to our algorithms is available at the python level or at the C level.
+    The python interface is documented through the \msprime{} documentation
+    (but you need to select ``latest'': \url{https://msprime.readthedocs.io/en/latest/});
+    and closely mirrors the underlying C library.
+    We have also provided a link to the  developing set of tutorials 
+    at \url{https://github.com/tskit-dev/tutorials} \llname{ll:tutorials}.
+}
+
+\begin{point}{}
+Figure 1 shows that pedigree recording seems to scale poorly in N. Going from
+N=1e3 to N=1e4 results in what seems like a $\sim 5 \times$ increase in runtime, but going
+up to N=5e4 results in a $> 10 \times$ increase in runtime. Given your attention to
+scaling, is this expected?
+\end{point}
+
+\reply{
+    This is actually the fault of a quadratic algorithm for counting mutations
+    under the hood in \fwdpp{}.
+    In fact, some of us (KRT) are currently using ideas from tree sequences to fix this bottleneck.
+}
+
+%%%  Minor comments:
+
+\begin{point}{}
+The word "thanks” in the discussion of coalescent caveats is awkward.
+\end{point}
+
+\reply{
+    Thanks; fixed. \revref
+}
+
+\begin{point}{}
+The Appendices seem to be out of order with the text. The current order does
+not seem any more logical.
+\end{point}
+
+\reply{
+    Good point. Fixed.
+}
+
+\begin{point}{}
+In Figure 1, it would potentially be helpful to see y-axis on a log scale so
+that we could see scaling on the low end. Legend for this figure refers to
+``right column of panels''. How many threads were used to obtain the dashed line?
+\end{point}
+
+\reply{
+    Thanks for the suggestions: we've switched the figure to a log scale,
+    and noted the number of threads (four).
+    The speedup is not that impressive for a variety of reasons, though, which
+    are hard to tease apart, so we haven't elaborated on this point.
+    (But, for the reviewer's interest: false sharing is one possible
+    issue, but perhaps $5 \times 10^4$ diploids may be too few to make
+    parallelization ``worth'' it when spawning new threads each generation.)
+}
+
+\begin{point}{}
+``its{'}'' on page 11.
+\end{point}
+
+\reply{
+    Thanks. Fixed, in two places.
+}
+
+\begin{point}{}
+I appreciate the ``how does it perform in theory?'' section, but the final note
+on the use of ``nedigree'' seems unnecessary. If you are going to use it (which I
+do not recommend), you can justify it, otherwise drop it.
+\end{point}
+
+\reply{
+    We have removed this section.
```

Comment:
Sad.

Reply:
Dunno.  I like it, but it may be that the days of whimsical sections of scientific papers are over.

Reply:
Only if we let them end... =)

Reply:
let's keep it! if doing so is a bar to publication the field has bigger problems and we should all change jobs

Reply:
if we do keep it perhaps we should just mention nedigree earlier, in the intro where we introduce this concept to make it more integral? i also don't see what 'justify' could mean for a new term, beyond what we already had there re etymology...

Reply:
hm, but where?

Reply:
I think the reviewer did mean that we should use it in the text if we keep it, but I'm not sure we want to do that, since really we're using "tree sequence" in its place.

Reply:
Hm. Good point. Well a tree sequence is just one representation of a nedigree? Could mention this upfront

Reply:
> we debated a number of possible terms for the embellished pedigree,
> i.e., the ``pedigree with ancestral recombination information'',
> the object through which each tree of a tree sequence is threaded.